ruby3.2-json-jwt.yaml

```diff
@@ -59,3 +59,8 @@ var-transforms:
     match: ^ruby(\d\.\d+)-.*
     replace: $1
     to: rubyMM
+
+test:
+  pipeline:
+    - runs:
+        gem-check
```

Comment:
I don't know how this works.
I think you want 
```yaml
test:
  pipeline:
    - uses: gem-check
```

Reply:
Crap, you're right.  Was working on this way too late last night...